migrate.go

```diff
@@ -15,7 +15,7 @@ import (
 var (
 	upMatcher    = regexp.MustCompile(`^([\w-]+).up.sql$`)
 	downMatcher  = regexp.MustCompile(`^([\w-]+).down.sql`)
-	tableMatcher = regexp.MustCompile(`^[\w]+$`)
+	tableMatcher = regexp.MustCompile(`^[\w\.]+$`)
```

Comment:
Inside `[]` you can actually just use the `.`, without escaping with `\`: https://regex101.com/r/I57CrD/1

Reply:
Hi @markuswustenberg done